database/models/receiptItem.model.ts

```diff
@@ -0,0 +1,62 @@
+import {
+    AllowNull,
+    AutoIncrement,
+    BelongsTo,
+    Column,
+    DataType,
+    ForeignKey,
+    Model,
+    PrimaryKey,
+    Table,
+} from 'sequelize-typescript';
+
+import Receipt from './receipt.model';
+import Store from './store.model';
+import StoreItem from './storeItem.model';
+
+@Table({
+    timestamps: true,
+})
+export default class ReceiptItem extends Model<ReceiptItem> {
+    @PrimaryKey
+    @AutoIncrement
+    @AllowNull(false)
+    @Column(DataType.INTEGER)
+    public pk: number;
+
+    @ForeignKey(() => Store)
+    @AllowNull(false)
+    @Column(DataType.INTEGER)
+    public store_pk: number;
+
+    @ForeignKey(() => StoreItem)
+    @AllowNull(false)
+    @Column(DataType.INTEGER)
+    public storeItem_pk: number;
+
+    @ForeignKey(() => Receipt)
+    @AllowNull(false)
+    @Column(DataType.INTEGER)
+    public receipt_pk: number;
+
+    @AllowNull(false)
+    @Column(DataType.STRING)
+    public item_name: string;
```

Comment:
```suggestion
    public name: string;
```
receiptItem 의 name임.